content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -437,6 +437,19 @@ Used on: PersistentVolumeClaim
 
 This annotation will be added to dynamic provisioning required PVC.
 
+### pv.kubernetes.io/bound-by-controller
+
+Example: `pv.kubernetes.io/bound-by-controller: "true"`
+
+Used on: PersistentVolume or PersistentVolumeClaim
+
+This annotation applies to {{< glossary_tooltip text="PersistentVolumes" term_id="persistent-volume" >}}
+(PVs) and {{< glossary_tooltip text="PersistentVolumeClaims" id="persistent-volume-claim" >}} (PVCs).
+When this annotation is present, that indicates that thWhee binding (either PV→PVC or PVC→PV) was installed
```

Comment:
Hi @afzal442 i am curious on the term `thWhee binding` could you let me know  whether it  is a typo or reference to it.

Reply:
good catch @Kartik494! now look!

Reply:
It's look fine from the previous changes.